_posts/2024-10-23-cockpit-327.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Cockpit 327
+author: martinpitt
+date: '2024-10-23'
+tags: cockpit
+slug: cockpit-327
+category: release
+summary: 'connect to similar servers without Cockpit installed'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 327:
+
+
+## Connect to similar servers without Cockpit installed
+
+The [Cockpit Client flatpak](https://flathub.org/apps/org.cockpit_project.CockpitClient) has been capable of connecting to remote machines without Cockpit installed since its beginning in [release 295](https://cockpit-project.org/blog/cockpit-295.html). Recently in version [326](https://cockpit-project.org/blog/cockpit-326.html), the [cockpit/ws container](https://quay.io/repository/cockpit/ws) got the same feature.
+
+With this release Cockpit gradually introduces this feature to the standard Linux distribution packages (rpm/deb/Arch). Initially you can connect to a remote machine without installed Cockpit if it runs the same operating system version as the host from which you make the connection. This ensures compatibility and safety:  The flatpak and ws container can be assumed to be always up to date in production, and hence can connect to any OS. But distribution packages can be quite old, and thus not be compatible when connecting to newer OS versions.
```

Comment:
What if we lead with the benefit, like this, instead of background info first?

```suggestion
Support for connecting to remote machines without Cockpit installed now extends to standard Linux distribution packages: RPM, DEB, and Arch

The [Cockpit Client flatpak](https://flathub.org/apps/org.cockpit_project.CockpitClient) has been capable of connecting to remote machines without Cockpit installed since its beginning in [release 295](https://cockpit-project.org/blog/cockpit-295.html) and the [cockpit/ws container](https://quay.io/repository/cockpit/ws) since [version 326](https://cockpit-project.org/blog/cockpit-326.html) recently added this functionality.

To ensure compatibility and safety, package-installed versions of Cockpit only allow connections to remote machines without Cockpit running the same operating system version. Future releases may relax this limitation.
```

Reply:
Sounds good! I'll add a period to the first paragraph, though.

Reply:
I also dropped the RPM/DEB/Arch thing -- "standard Linux distribution packages" is already describing this well enough.